src/databricks/labs/remorph/install.py

```diff
@@ -21,80 +20,145 @@
+            raise err
+        return configs
+
+    def configure(self) -> RemorphConfigs:
+        """
+        Returns the MorphConfig If it exists on the Databricks Workspace,
+         else prompts for the new Installation
+        :return:
+        """
+        morph_config = None
+        reconcile_config = None
 
-    def configure(self) -> MorphConfig:
         try:
-            return self._installation.load(MorphConfig)
+            morph_config = self._installation.load(MorphConfig)
         except NotFound as err:
-            logger.debug(f"Cannot find previous installation: {err}")
-        logger.info("Please answer a couple of questions to configure Remorph")
+            logger.debug(f"Cannot find previous `transpile` installation: {err}")
+        # except (PermissionDenied, SerdeError, ValueError, AttributeError): logger.warning(f"Existing installation
+        # at {self._installation.install_folder()} is corrupted. Skipping...")
 
-        # default params
-        catalog_name = "transpiler_test"
-        schema_name = "convertor_test"
-        ws_config = None
+        try:
+            reconcile_config = self._installation.load(ReconcileConfig)
+        except NotFound as err:
+            logger.debug(f"Cannot find previous `reconcile` installation: {err}")
+        # except (PermissionDenied, SerdeError, ValueError, AttributeError): logger.warning(f"Existing installation
+        # at {self._installation.install_folder()} is corrupted. Skipping...")
 
-        source_prompt = self._prompts.choice("Select the source", list(SQLGLOT_DIALECTS.keys()))
-        source = source_prompt.lower()
+        if morph_config or reconcile_config:
+            return RemorphConfigs(morph_config, reconcile_config)
 
-        skip_validation = self._prompts.confirm("Do you want to Skip Validation")
+        return self._configure_new_installation()
 
-        if not skip_validation:
-            ws_config = self._configure_runtime()
-            catalog_name = self._prompts.question("Enter catalog_name")
-            try:
-                self._catalog_setup.get(catalog_name)
-            except NotFound:
-                self.setup_catalog(catalog_name)
+    def _save_and_open_config(self, module: str, config: MorphConfig | ReconcileConfig):
+        logger.info(f"Saving the ** {module} ** configuration in Databricks Workspace")
+        self._installation.save(config)
+        ws_file_url = self._installation.workspace_link(config.__file__)
+        if self._prompts.confirm(f"Open `{config.__file__}` in the browser and continue...?"):
+            webbrowser.open(ws_file_url)
 
-            schema_name = self._prompts.question("Enter schema_name")
+    def _configure_new_installation(self) -> RemorphConfigs:
+        """
+        Prompts for the new Installation and saves the configuration
+        :return: MorphConfig
+        """
+        morph_config, reconcile_config = self._install.prompt_for_new_installation()
```

Comment:
it's a bit clunky to read this. what if we add one more module to configure?.. then refactoring will be bigger. use `choice_from_dict` prompt to simplify:

```
function_to_call = prompts.choice_from_dict('Select a module to cofigure', {
    'reconcise': self._reconcile_prompts,
    'transpile': self._transpile_prompts
})
function_to_call()
```

https://github.com/databrickslabs/blueprint?tab=readme-ov-file#single-choice-from-list

Reply:
Done, updated the code